atlp-rwanda/phantom-fe-elite
# #28 Feature:  Implementing landing-page.

What does this PR do?

- Create UI for landing-page

Description of Task to be completed?

- Landing Page UI is created. It includes three components; footer, navbar and LandingPage.  

- It should be responsive.

- It should perfectly resemble to the corresponding design in the mockup

-  UI skeleton for loading displayed

How should this be manually tested?
- Clone the repo
- checkout to branch ft-landing-page-28
- pull the changes by git pull
- install all packages by running npm install
- start app by running npm run start.
-  It should dispaly all contents of the landing page as depicted in the design

Any background context you want to provide?

- [ Work In Progress
 ]; 

What are the relevant pivotal tracker stories?
[Trello Board # 28](https://trello.com/c/7MYeedeR/28-landing-page)

Screenshots (if appropriate)
